socorro/cron/jobs/ftpscraper.py

```diff
@@ -313,11 +329,7 @@ def _scrape_json_releases_and_nightlies(
 
     def _insert_build(self, cursor, *args, **kwargs):
         self.config.logger.debug('adding %s', args)
-        if self.config.dry_run:
-            print "INSERT BUILD"
-            print args
-            print kwargs
-        else:
```

Comment:
They both print `args` one with `config.logger` and one with `print`. Feel free to add whatever more extra you want when you really need to do the comparison.